devtools/conda-envs/docs_env.yaml

```diff
@@ -7,14 +7,30 @@ dependencies:
   # Base depends
   - python
   - pip
-  - pydantic
-  - pint
-  - openff-toolkit-base >=0.11.3
-  - openff-models >=0.0.4
+  - numpy >=1.21
+  - pydantic <2.0.0a0
   - openmm >=7.6
+  - openff-toolkit >=0.11.3
+  - openff-models >=0.0.4
+  - openff-nagl
```

Comment:
```suggestion
  - openff-toolkit >=0.13
  - openff-models >=0.0.4
```

Reply:
I'm probably reviewing my own changes here, the test environment is a pain in the rear to simultaneously keep up to date and minimal to the use case